cmd/cli/README.md

```diff
@@ -18,7 +18,7 @@ The program accepts a number of flags which are documented in the table below:
 |    `-u`, `--url`     |  3scale Admin Portal URL                                                |   Yes   |         |
 |    `-s`, `--service` |  3scale API/Service ID                                                  |   Yes   |         |
 |    `--uid`           |  Unique UID for resource names. Derived from URL and service  if unset  |   No    |         |
-|    `--fixup`         |  Try to automatically fix validation errors                             |   No    | False   |
+|    `--fixup`         |  Try to automatically fix validation errors. Autoenabled if --uid unset |   No    | False   |
```

Comment:
Not part of this but might aswel fix when touching this file. Extra space in front of "if" on previous line

Reply:
:+1: